### ISSUE TITLE
Triggering by ocular trauma

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9658]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

PMID: 30081702 Sympathetic ophthalmia (SO) is a bilateral, granulomatous panuveitis that occurs after ocular trauma or
surgical procedure.
PMID: 30959067 Sympathetic ophthalmia (SO) is defined as a bilateral granulomatous uveitis occurring after penetrating ocular trauma or intraocular surgery in 1 eye
PMID: 35647958 Sympathetic ophthalmia is a rare, bilateral, granulomatous, panuveitis following penetrating trauma or
surgery to one eye. 
PMID: 30207811 Sympathetic ophthalmia is a rare, bilateral, granulomatous, panuveitis following penetrating trauma or
surgery to one eye. 
PMID: 33256432 Table 3. Triggering events in patients with SO: trauma 100%

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)** 
Sympathetic ophthalmia Orpha 79098

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000711
- label: Triggering by ocular trauma
- parents: [Triggered by physical trauma (HP:0031135)](https://hpo.jax.org/app/browse/term/HP:0031135)
- pmid: PMID:30081702|PMID:35647958
- def: Applies to a sign or symptom that is provoked or brought about by trauma to the eye.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9658
- synonyms:
